src/displayapp/LittleVgl.h

```diff
@@ -60,9 +61,9 @@ namespace Pinetime {
       uint16_t writeOffset = 0;
       uint16_t scrollOffset = 0;
 
-      uint16_t tap_x = 0;
-      uint16_t tap_y = 0;
+      lv_point_t touchPoint = {0};
```

Comment:
I think this should be initialized with `{}`, otherwise I think that only the `x` variable is set to `0`, the other variable `y` is undefined. I'm not sure, this should be tested